ratatool-sampling/src/test/scala/com/spotify/ratatool/samplers/BigSamplerTest.scala

```diff
@@ -283,8 +297,11 @@ object BigSamplerTest extends Properties("BigSampler") {
             )(_.getRecord(r).set(f))
         (i, tbGen.sample.get, s"$r.$f")
     }) { case (avro, tblRow, f) =>
-    val tblHash = BigSampler.hashTableRow(tblRow, f, tblSchemaFields, newTestKissHasher()).hash()
-    val avroHash = BigSampler.hashAvroField(avro, f, avroSchema, newTestKissHasher()).hash()
+    val tblHash = BigSampler.hashTableRow(tblRow, f, tblSchemaFields,
+      newTestKissHasher(), ByteHasher.standardHash).hash()
```

Comment:
More of a style thing I guess. It seems odd to me to have to pass something specifically for one field format (bytes) in this signature

Reply:
We can discuss some more